apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -201,6 +218,20 @@ private void sendMessage(JSONObject json) {
     log.debug(String.format("Sending Message: {} to {}", msg, TOPIC_ARN));
     PublishResult publishResult = snsClient.publish(publishRequest);
     // TODO: check on size of message and truncation etc (this can come later if/when we add more)
-    log.debug("Published SNS Message - " + publishResult.getMessageId());
+    log.info("Published SNS Message - " + publishResult.getMessageId());
   }
+
+  private Map<String, String> getHkaasParams(Map<String, String> tableParameters) {
+    Map<String, String> hkaasParams = new HashMap<>();
+    if (!StringUtils.isEmpty(tableParamFilter)) {
```

Comment:
we check for isEmpty on line 78, if we cannot use the empty Pattern you create there, let's set it to null just check the pattern in this method.

Reply:
good suggestion. Done.